src/backy/main.py

```diff
@@ -9,6 +9,7 @@
 import humanize
 import structlog
 import tzlocal
+import yaml as yaml_
```

Comment:
I'd prefer if we'd use `yaml` here and use `dest='yaml_'` for the argparse argument.